### PR TITLE
Create stable.js

### PR DESCRIPTION
**Pull request by d2lam**
Stjohn's script for tagging and pushing stable image

### CODE REVIEW

File: stable.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ev
```
Comment (FenrirUnbound) [inline stable.js]: What do the `-ev` flags change?

**Comment by d2lam** [comment]
@FenrirUnbound  oopsie, changed to `.sh`

File: stable.js (hunk 1 above)
Comment (d2lam) [inline stable.js]: -v for verbose so it would print out the input lines as it goes
-e for exits immediately if fails

Looked this up :p

**Comment by d2lam** [comment]
@tkyi  added README

File: README.md [hunk 2]
```diff
@@ -10,3 +10,10 @@ This script will create all pipelines and their corresponding secrets, except fo
 $ npm install
 $ node mass-create-pipelines.js
 ```
+
+## To tag stable image:
+Run `stable.js`. For example, to retag latest UI as stable:
```
Comment (FenrirUnbound) [inline README.md]: `stable.sh` 😝

Comment (d2lam) [inline README.md]: UGHHHH >.<

**Pull request merged**